eng/native/build-commons.sh

```diff
@@ -289,6 +289,17 @@ else
   fi
 fi
 
+if [[ "$platform" == "Linux" && "$__NumProc" == "1" ]]; then
```

Comment:
nit: for consistency, we can move it after line 286 (above `else\n__NumProc=1` block) and use same single-line condition which we used here https://github.com/dotnet/runtime/blob/0dd6b7be33b9a3bd5953bceff07377839af92a5c/eng/native/init-os-and-arch.sh#L45

Reply:
Thanks!